src/luos.c

```diff
@@ -928,3 +952,27 @@ void Luos_Flush(void)
 {
     Robus_Flush();
 }
+
+/******************************************************************************
+ * @brief register a new package
+ * @param package to register
+ * @return None
+ ******************************************************************************/
+void Luos_AddPackage(void (*Init)(void), void (*Loop)(void))
+{
+    package_table[package_number].Init = Init;
+    package_table[package_number].Loop = Loop;
+
+    package_number += 1;
+}
+
+/******************************************************************************
+ * @brief Create a service to signal empty node
+ * @param None
+ * @return None
+ ******************************************************************************/
+void Luos_EmptyNode(void)
+{
+    revision_t revision = {.unmap = {1, 0, 0}};
```

Comment:
Because the EmptyNode is a void container maintained by Luos, we should reuse the Luos revision as revision here.

Reply:
OK, I make this change.

Reply:
alternatively, you can just click on the "Resolve conversation" button.